src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -142,6 +147,10 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             services.AddSingleton<IFunctionMetadataManager, FunctionMetadataManager>();
             services.AddSingleton<IWebFunctionsManager, WebFunctionsManager>();
             services.AddHttpClient();
+
+            // Named HttpClient for Extension Bundle downloads with Polly-based retry (no AutomaticDecompression per requirement)
```

Comment:
This comment details the internal implementation details of the method. If you want to capture that, please add that to the method docs, not the caller.

Reply:
Make sense. Removed.